packages/google_maps_flutter/google_maps_flutter/ios/google_maps_flutter.podspec

```diff
@@ -3,7 +3,7 @@
 #
 Pod::Spec.new do |s|
   s.name             = 'google_maps_flutter'
-  s.version          = '0.0.1'
+  s.version          = '0.0.2'
```

Comment:
Don't edit this version, it's just a stand in and doesn't mean anything.  `pub` has already handled the versioning solving at the point this podspec is being used.

Reply:
Done! PTAL!